bindings/go/constructor/options.go

```diff
@@ -101,3 +105,18 @@ const (
 	// ComponentVersionConflictSkip will skip the construction of the component version if it already exists in the target repository.
 	ComponentVersionConflictSkip
 )
+
+// ExternalComponentVersionCopyPolicy defines the policy for handling external component version references
+// when interacting with the target repository.
+// If the constructor library encounters an external component version reference that is not part of the constructor
+// specification, it will use this policy to determine how to handle the reference.
+type ExternalComponentVersionCopyPolicy int
+
+const (
+	// ExternalComponentVersionCopyPolicySkip will skip the copy of the component version to the target repository.
+	// This is the default policy.
+	ExternalComponentVersionCopyPolicySkip = iota
```

Comment:
is this the default behavior? I would expect iota 0 to be default

Reply:
yes that would be default, as in by default external references are not copied.